frontend/src/components/object/ObjectTable.vue

```diff
@@ -203,7 +203,8 @@ const filters = ref({
           <InputSwitch
             v-model="data.public"
             :disabled="!permissionStore.isObjectActionAllowed(
-              data.id, getUserId, Permissions.MANAGE, props.bucketId as string)"
+              data.id, getUserId, Permissions.MANAGE, props.bucketId as string) ||
+              !usePermissionStore().isUserElevatedRights()"
```

Comment:
I'm half tempted to recommend applying the De Morgan's Law inverse to this boolean statement in order to improve readability, as this has a lot of negative assertions at the moment. I also suspect that if we evaluate `isUserElevatedRights` first instead of `isObjectActionAllowed`, we can take advantage of short circuiting and potentially reduce one network call.
```ts
:disabled="!(
  usePermissionStore().isUserElevatedRights() && 
  permissionStore.isObjectActionAllowed(
    data.id, getUserId, Permissions.MANAGE, props.bucketId as string))"
```

Reply:
I think what makes it so confusing / hard to read is that the Vue prop itself being a negative itself means that we need some level of double negation in there - i.e. you *have* to tell Vue "do **not** enable this if...", instead of "**only** enable if...".

But I agree with you on all counts there.